fe1-web/src/features/rollCall/screens/RollCallScanner.tsx

```diff
@@ -63,6 +63,10 @@ const RollCallOpened = () => {
 
   // this is needed as otherwise the camera may stay turned on
   const [showScanner, setShowScanner] = useState(false);
+  // use a mutable ref to avoid the problem that QrCodeScanner's handle scan function
+  // is never changed, i.e. the bound variables stay the same during the scanner's lifetime
+  // this is due to a react-camera bug and is hopefully fixed at some point in the future
+  const attendeePopTokens = useRef(navigationAttendeePopTokens);
```

Comment:
does this behave correctly when switching between different roll calls, assuming multiple roll calls exist in the LAO?

Reply:
As far as I can tell it does, yes. Tested having 2 open roll calls, switching between them and closing the one created 2nd and this should be the only reachable scenario. Closing a non-latest roll call is impossible and you can only get to this screen when a roll call is open.